lib/tree_sitter/node.rb

```diff
@@ -163,5 +163,94 @@ def fetch(*keys)
       end
       fields.values_at(*keys)
     end
+
+    # Regex for line annotatin extraction from sexpr with source.
+    #
+    # @!visibility private
+    LINE_ANNOTATION = /@\{([^}]*)\}/
```

Comment:
I had used that syntax just as an example of what could be used, but maybe you should use something much less likely to occur.  Such as `\0{...\0}`, with genuine nul characters.